old-fashioned-image-build

```diff
@@ -177,6 +181,11 @@ time for FILE in `lxc exec lp-$SERIES-${ARCH} -- find /build -mindepth 1 \
     lxc file pull lp-$SERIES-${ARCH}$FILE $OUTPUT_DIRECTORY/
 done
 
+if [ $CLEANUP = false ] ; then
```

Comment:
My bash-fu is not powerful enough to know if `=` is ok here instead of `==` or `-eq`

Reply:
'man test' will help here, the '[' command is provided by test(1) and for string comparison you use '='.  We could also use the bash '[[' with the '==' string comparison but throughout the code the test evaluation is used rather than the [[ bash-ism.

Reply:
No boolean types in bash, got it.